dbcon/execplan/calpontsystemcatalog.h

```diff
@@ -224,7 +224,7 @@ class CalpontSystemCatalog : public datatypes::SystemCatalog
     bool autoincrement = 0;  // set to true if  SYSCOLUMN autoincrement is �y�
     uint64_t nextvalue = 0;  // next autoincrement value
     uint32_t charsetNumber = default_charset_info->number;
-    const CHARSET_INFO* cs = nullptr;
+    const mutable CHARSET_INFO* cs = nullptr;
```

Comment:
Are you sure we need a trick like this to make `getCharset` const? We actually changes the state of ColType instance